docs/topics/idioms.md

```diff
@@ -49,6 +51,8 @@ if ("jane@example.com" !in emailsList) { ... }
 println("Name $name")
 ```
 
+Learn the difference between [Java and Kotlin string concatenation](java-to-kotlin-idioms-strings.md#concatenate-strings).
```

Comment:
I wonder if I can call concatenation so here because here the section is called "String interpolation".

Reply:
@koshachy what do you think?

Reply:
LGTM